CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+# dbt_shopify_source v0.13.0
+
+[PR #91](https://github.com/fivetran/dbt_shopify_source/pull/91) includes the following changes:
+## Under the Hood
```

Comment:
```suggestion
## Breaking Changes
```
I would call this or at least part of it "breaking changes" since we bumped the minor version and have no other changes.